action.yml

```diff
@@ -42,7 +42,20 @@ runs:
       run: |
         git clone --depth 1 ${{ inputs.repository_url }} -b ${{ env.source_branch }} t3docsproject
 
+    - id: enable_guides
+      working-directory: t3docsproject
+      shell: bash
+      run: test -e guides.xml && echo "GUIDES=true" || echo "GUIDES=true" >> "GITHUB_OUTPUT"
+
+    - uses: TYPO3-Documentation/render-guides@main
+      id: render-guides
+      if: steps.enable_guides.outputs.GUIDES == 'true'
+      with:
+        input: t3docsproject
+        output: RenderedDocumentation
+
     - id: jobfile
+      if: steps.enable_guides.outputs.GUIDES == 'false'
```

Comment:
Can GUIDES be false? It is only set to true above. Shouldn't it a `GUIDES != 'true'`?

Reply:
You are right, I made a mistake in the detection step